pkg/manager/manager.go

```diff
@@ -72,6 +73,7 @@ func New(opts Options) (Manager, error) {
 	}
 
 	// Build the controller manager.
+	klog.Infof("Making new manager from options: %v", opts)
```

Comment:
This needs to be redacted. We absolutely cannot print credentials to the log.

Reply:
yup, fixed